src/de.rs

```diff
@@ -2015,6 +2015,21 @@ impl Error {
         self.inner.line.map(|line| (line, self.inner.col))
     }
 
+    /// Get the message associated with this error
+    pub fn message(&self) -> &str {
+        &self.inner.message
+    }
```

Comment:
This seems a little off, because this only returns the message for the custom kind.  I'm kinda wondering why the message isn't part of `ErrorKind::Custom`.

Reply:
You mean using the `message` name for this function is weird? I am happy to use `custom_message` instead. 

Moving the message to `ErrorKind::Custom` also works, although it would be a slightly larger change.

Reply:
Yea, ideally the message would be embedded in `ErrorKind::Custom(String)`, and remove this method.  It just seems a little too odd to have a method like this (and a dedicated `message` field) for one kind.